schema.sql

```diff
@@ -61,4 +61,26 @@ CREATE TABLE visits (
     date DATE NOT NULL,
     FOREIGN KEY(vets_id) REFERENCES vets(id) ON DELETE RESTRICT ON UPDATE CASCADE,
     FOREIGN KEY(animals_id) REFERENCES animals(id) ON DELETE RESTRICT ON UPDATE CASCADE
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animals_id = 4
+SELECT * FROM visits WHERE vets_id = 2;
+SELECT * FROM owners WHERE email = 'owner_18327@mail.com';
+
+-- Way to decrease the time exec for the fisrt query
+CREATE INDEX animals_id_index ON visits(animals_id);
+SELECT COUNT(*) FROM visits WHERE animals_id = 4
+
+-- Way to improve execution time of the other two queries.
+CREATE INDEX visits_id_index ON visits(vets_id);
+SELECT * FROM visits WHERE vets_id = 2;
+
+-- CREATE INDEX owners_id_index ON owners(email);
```

Comment:
Please add `INDEX ON owners` here as it's the proposed solution. You are only selecting but not adding index on foreign key.

Reply:
Okay thanks